test/azure/src/main/java/fixtures/azureparametergrouping/implementation/ParameterGroupingsImpl.java

```diff
@@ -120,7 +119,7 @@ public void postRequired(ParameterGroupingPostRequiredParameters parameterGroupi
      *
      * @param parameterGroupingPostRequiredParameters Additional parameters for the operation
      * @throws IllegalArgumentException thrown if parameters fail the validation
-     * @return a {@link Single} emitting the RestResponse<Void, Void> object
+     * @return the {@link Single&lt;RestResponse&lt;Void, Void&gt;&gt;} object if successful.
```

Comment:
Is this `&lt;` stuff needed for the Javadoc to render properly?

Reply:
I believe so. We were doing it in some places and not in others. The benefit of the simplifications that I'm doing is that if we decide it's not necessary, then we'll have far fewer places to change.